src/appengine/handlers/cron/project_setup.py

```diff
@@ -881,17 +876,19 @@ def get(self):
       logs.log_error('Failed to get AFL Fuzzer entity.')
       return
 
-    # TODO(ochang): Move hardcodes to config.
+    bucket_config = local_config.ProjectConfig().sub_config(
+        'project_setup.build_buckets')
+
```

Comment:
We should probably bail out if this is None. or leave a log_error with bailout

Reply:
Will address in another PR.